### DISCUSSION TITLE
Auto fill column 'Created Date'

### DISCUSSION THREAD

**Discussion by farisdurrani** [community/community#45641] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

I wish the Projects board view has a new column, or at least a new Workflow automation, that has the Created Date auto-populated from the Issues. This makes it easier for us to order by the age of the Issues

**Comment by rileybroughten** [comment, marked as answer]
Hello and thanks for the feedback @farisdurrani! 👋 Understood how these types of issue timestamps as fields would make it easier to sort and customize your project view. I would suggest following along in the main discussion post [here](https://github.com/community/community/discussions/8518) as we will be providing updates on that issue once that works kicks off!
